packages/server-utils/src/constructExportEmail.ts

```diff
@@ -28,13 +17,13 @@ type ResponseBody = {
 
 type Req = {
   query: {
-    emailExportFileMode?: EmailExportFileModes;
+    platform?: 'tupaia' | 'adminPanel' | 'datatrak';
   };
 };
 
 export const constructExportEmail = async (responseBody: ResponseBody, req: Req) => {
-  const { emailExportFileMode = EmailExportFileModes.DOWNLOAD_LINK } = req.query;
   const { error, filePath } = responseBody;
+  const { platform } = req.query;
   const subject = 'Your export from Tupaia';
```

Comment:
I wonder if the subject should change depending on the platform?

Reply:
Yes, good idea